docs/Tutorial.md

```diff
@@ -10,7 +10,7 @@ Admin-on-REST uses React. We'll use Facebook's [react-create-app](https://github
 npm install -g create-react-app
 create-react-app test-admin
 cd test-admin/
-npm install --save-dev admin-on-rest
+npm install --save-dev babel-runtime admin-on-rest
```

Comment:
No, the PR #9 aims the project `package.json`. Once it's published, you won't need to install `babel-runtime` by hand, since it'll be a prod dependency.